src/libtest/lib.rs

```diff
@@ -1155,19 +1200,40 @@ where
     Ok(())
 }
 
+fn parse_positive_number(number_str: &str, variable_name: &str) -> usize {
+    let number: Option<usize> = number_str.parse().ok();
```

Comment:
Inline this `let`, use `.parse::<usize>()`. Also, `.ok()` is redundant, just match on `Ok(n)` instead.